### PR TITLE
certsuite: process flags refactor

### PR DESCRIPTION
**Pull request by jmontesi**

### CODE REVIEW

**Review by sebrandon1 — APPROVED** [review]

**Review by bnshr — CHANGES_REQUESTED** [review]

File: pkg/certsuite/certsuite.go [hunk 1]
```diff
@@ -141,12 +145,11 @@ func Run(labelsFilter, outputFolder string) error {
...
 	claimBuilder.Build(claimOutputFile)
 
-	// Write the file to the flag output-dir
-	outputFile := *flags.OutputDir + "/" + junitXMLOutputFile
-
+	// Create JUnit file if required
 	if configuration.GetTestParameters().EnableXMLCreation {
-		log.Info("XML file creation is enabled. Creating JUnit XML file: %s", outputFile)
-		claimBuilder.ToJUnitXML(outputFile, startTime, endTime)
+		junitOutputFile := filepath.Join(outputFolder, junitXMLOutputFileName)
```
[the remark this answers is not in the document]
Comment (jmontesi) [reply pkg/certsuite/certsuite.go]: Done

File: pkg/certsuite/certsuite.go [hunk 2]
```diff
@@ -71,45 +73,47 @@ func getK8sClientsConfigFileNames() []string {
...
-		if err != nil {
-			return fmt.Errorf("could not list test cases, err: %v", err)
-		}
-		cli.PrintChecksList(checksIDs)
-
-		os.Exit(1)
-	}
-
 	fmt.Println("Running discovery of CNF target resources...")
 	fmt.Print("\n")
```
[the remark this answers is not in the document]
Comment (jmontesi) [reply pkg/certsuite/certsuite.go:118]: I wanted to be more explicit about the fact that I need an extra new line there.

[the remark this answers is not in the document]
Comment (bnshr) [reply pkg/certsuite/certsuite.go:118]: Well, then you could add \n in the previous line if it is about nice display related topic. This line is unnecessarily piqued my interest.

[the remark this answers is not in the document]
Comment (jmontesi) [reply pkg/certsuite/certsuite.go:118]: Our linter disagrees: `printf: "fmt.Println" arg list ends with redundant newline (govet)`

**Comment by bnshr** [comment]
Merge it!

**Review by bnshr — APPROVED** [review]

**Pull request merged**